app/default/content_routes.py

```diff
@@ -30,9 +30,16 @@ def all_questions(fund_short_name, round_short_name):
     round = get_round_data_by_short_names(fund_short_name, round_short_name)
     if not round:
         return abort(404)
-    return render_template(
-        "cof_r2_all_questions.html", round_title=round.title
-    )
+
+    if fund_short_name.lower() == "cof":
+        if "r2" in round_short_name.lower():
+            return render_template(
+                "cof_r2_all_questions.html", round_title=round.title
+            )
+        elif "r3" in round_short_name.lower():
+            return render_template(
+                "cof_r3_all_questions.html", round_title=round.title
+            )
 
```

Comment:
Instead of if statements can we just construct the page name from the fund and round? Otherwise we'll have to update this function every time we add a round. So:
`render_template(f"{fund_short_name}_{round_short_name}_all_questions.html")`

Reply:
Good Shout. Done now @srh-sloan